seriesLength.py

```
@@ -1,143 +1,144 @@
+    # Sort durations by value (shortest series first)
+    sorted_durations = dict(sorted(durations.items(), key=lambda x: x[1]))
+    
+    # Display results in console
+    logging.info("\nSeries sorted by total duration:")
+    total_duration = 0
+    for series_name, duration in sorted_durations.items():
+        formatted = format_duration(duration)
+        logging.info(f"{series_name}: {formatted}")
+        total_duration += duration
+    
+    logging.info(f"\nOverall Statistics:")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)